modP2pImpl/src/org/aion/p2p/impl1/P2pMgr.java

```diff
@@ -86,6 +86,8 @@
 
     public static final Logger p2pLOG = AionLoggerFactory.getLogger(LogEnum.P2P.name());
 
+    public static final int WORKER = 32;
```

Comment:
Why 32?

Reply:
it's by the design of the method TaskSend.hash2Lane()